packages/base-styles/_variables.scss

```diff
@@ -55,16 +55,16 @@ $radius-round: 50%;     // For circles and ovals.
  */
 
 // For sections and containers that group related content and controls, which may overlap other content. Example: Preview Frame.
-$elevation-x-small: 0 0.7px 1px rgba($black, 0.1), 0 1.2px 1.7px -0.2px rgba($black, 0.1), 0 2.3px 3.3px -0.5px rgba($black, 0.1);
+$elevation-x-small: 0 1px 1px rgba(0,0,0, 0.03), 0 1px 2px rgba(0,0,0, 0.02), 0 3px 3px rgba(0,0,0, 0.02), 0 4px 4px rgba(0,0,0, 0.01);
```

Comment:
What's the reason for switching from `$black` to `0, 0, 0` ?

Reply:
Oh, I was copy / pasting between `_variables.scss` and `config-values.js`. `$black` doesn't work in the latter :)

I'll revert to `$black` once we're happy with the values.